rfcs/0102-clarify-strategy-decision-process.md

```diff
@@ -0,0 +1,39 @@
+Authors: @tqchen
+
+- Feature Name: [Process RFC] Clarify Community Strategy Decision Process
+- Start Date: 2023-08-03
+- RFC PR: [apache/tvm-rfcs#0102](https://github.com/apache/tvm-rfcs/pull/0102)
+- GitHub Issue: [apache/tvm#0000](https://github.com/apache/tvm/issues/0000)
+
+## Summary
+
+Machine Learning Compilation (MLC) is an emerging field in fast development.
+With the tremendous help from the whole community, it’s exciting to see that TVM delivers significant needs from and to
+developers and thus has become widely popular in both academia and industry.
+
+As the community pushes for different goals that help each other, naturally, there
+are strategy decision points about overall directions and new modules adoptions.
+These decisions are not fine-grained code-level changes but are important for a
+community to be viable in the long term.
+The process of bringing those changes is less clarified to the community, and hurdles can be high.
+We have made attempts in the past to bring a more verbose processes, but this has proven to be less successful.
+One observation is that it is hard for broader volunteer developers and community members to follow complicated processes.
+Additionally, different members can have different interpretations of how to do things,
+leading to stagnation and lack of participation from volunteer members.
+
+We are in a different world now in the case of ML/AI ecosystem, and it is critical for
+the community to be able to make collective decisions together and empower the community.
+Following the practices of existing ASF projects (e.g. hadoop), we propose to use a simple process for strategic decisions.
+
+## Proposal: Strategy Decision Process
+
+We propose the following clarification of the strategy decision process:
+It takes lazy 2/3 majority (at least 3 votes and twice as many +1 votes as -1 votes)
+of binding decisions to make strategic decisions in the TVM community, including:
+
+- Adoption of a guidance-level community strategy to enable new directions or overall project evolution.
+- Establishment of a new module in the project.
```

Comment:
Establishment of new modules in the project is the main subject of the very detailed (ongoing) RFC https://github.com/apache/tvm-rfcs/pull/95, so I'm wondering what's the strategy here.

Do we want to close that RFC as rejected and expand this one, or remove this point from here and continue the discussion on https://github.com/apache/tvm-rfcs/pull/95?

Reply:
Sorry, this response is not addressing the question.

Reply:
I think it is up to @Hzfengsy to decide what to do with the other RFC.   (addressing your question on what to do with existing RFC).  

I would remark that this reply is not taking the either the options that you listed, which contains specific operational ways of how. Instead it addresses the two action items:

- The other RFC: leave it up to @Hzfengsy , however i don't think any action needs to be taken until we conclude either one of the two.
- This RFC, I do not intend to expand on the particular item, as being stated, the goal is to is to come up with simple, less verbose process that everyone understand, as verbatim it is quite clear I think to many community members.

Reply:
I don't think the community benefits of intentionally having unwritten "less verbose" processes, but I understand from the message above that there is no intention of making the current text more clear, which is part defats the purpose of having an RFC process at all.

Reply:
Thank you for your inputs. The purpose of the RFC is indeed to collect the inputs, that includes yours as well as broader set of community members.

I understand that you do not agree the simple text is good, as you have made your point very clearly -- this comes down to disagreement on how to interpret clarity. 

Unfortunately one can not both expand the text(to fulfill one request) while also keep the text simple (to fulfill the need of others). After considering your input, as well as inputs from broader community members, I think it is good course of action to keep the proposal text as it is after looking at the inputs so far and let the community decide whether to accept it.

At any time point, you are more than welcomed to bring up a proposal to the community  as well that is independent from this one.